micrometer-support/src/main/java/io/javaoperatorsdk/operator/monitoring/micrometer/MicrometerMetrics.java

```diff
@@ -118,6 +168,12 @@ public void failedReconciliation(HasMetadata resource, Exception exception,
     return registry.gaugeMapSize(PREFIX + name + ".size", Collections.emptyList(), map);
   }
 
+  private List<Tag> gvkTags(Class<? extends HasMetadata> resourceClass) {
```

Comment:
There are several spots where GVK information is added to tags. Could this method be reused elsewhere? Also, it looks like sometimes it's prefixed with `resource`, sometimes not…

Reply:
`gauge` is a little different than others, since it is first needs to initialized. Then incremented or decremented. This is specially needed for that. But made it at least public static for now.

Reply:
>  Also, it looks like sometimes it's prefixed with resource, sometimes not…

what do you mean exactly?